contextual_services/contextual_services.model.lkml

```diff
@@ -2,3 +2,33 @@ connection: "bigquery-oauth"
 label: "Contextual Services"
 include: "//looker-hub/contextual_services/explores/*"
 include: "views/*"
+
+
+explore: +event_aggregates {
+  description: "Daily event rollups. This aggregated dataset is faster to query and contains a longer history."
+}
+
+explore: +quicksuggest_clicks {
+  description: "
+    The QuickSuggest click ping is sent when the user clicks on the QuickSuggest link.
+    It shares most of the fields as the QuickSuggest impression ping except that it doesn’t include the user search query, nor the matched keywords.
+    This data is kept for 30 days."
+}
+
+explore: +quicksuggest_impressions {
+  description: "
+    This impression ping is only sent on two conditions:
+    A user completes the search session in the URL bar, i.e. he or she navigates away from the search bar, or
+    a QuickSuggest link was shown in the URL bar at the end of the search session.
+    This means that any QuickSuggest link(s) that was displayed while the user was still typing would not trigger this impression ping.
+    The impression ping would only count the last QuickSuggest link if any when the search session ends.
+    This data is kept for 30 days."
```

Comment:
Could we change this to:

 A user completes the search session in the URL bar, i.e. he or she navigates away from the search bar, AND
 a QuickSuggest link was shown in the URL bar at the end of the search session.
 This means that any QuickSuggest link(s) that were displayed while the user was still typing would not trigger this impression ping.
This data is kept for 30 days."

Changes:

- Line 21: AND condition instead of OR
- Line 23: Small grammar fix
- Remove line 24: It's saying the same thing as line 23, but not resilient to the possibility that, in the future, we might show more than one QuickSuggest link at the same time.

Reply:
In my understanding of the two conditions, the second one is a subset of the first one, but I will double check with Nan.

Reply:
Confirmed from Nan that it's AND